Lexplorer/Shared/MainLayout.razor

```diff
@@ -27,7 +27,7 @@
         <NavMenu />
     </MudDrawer>
     <MudMainContent>
-        <MudContainer Fixed="true" Class="my-0 pt-0 extra-extra-extra-large">
+        <MudContainer Fixed="true" Class="mt-3 pt-0 extra-extra-extra-large">
```

Comment:
I remember it was my-3 originally, then you changed it back to my-0 - why? Shouldn't there a margin at the bottom as well? I have seen pages (I believe it was the accounts page) where the transaction table was the sudden end of the page and a margin would have been nice.

Reply:
I went back to an earlier version before my PR, and this was set to my-0.  
In my first PR (the enourmous one) I had changed it to my-3, but in the mobile one that got through I didn't do the change. I think this is the confusion.  
So it has been my-0 from before. And yes, it should be a margin on the top and bottom :) So that's why I added it here on the main container.